src/calculateRentalCost.js

```diff
@@ -29,6 +29,28 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const SumaOneDay = 40;
+  const discountShort = 20;
+  const DaysRangeShort = 3;
+  const DaysRangeLong = 7;
+  const discountLong = 50;
+  let ResultSuma = SumaOneDay * days;
```

Comment:
Variable names should be written in camelCase. 'ResultSuma' should be 'resultSum'.